src/login/logind-core.c

```diff
@@ -559,14 +559,14 @@ bool manager_is_lid_closed(Manager *m) {
         return false;
 }
 
-static bool manager_is_docked(Manager *m) {
+static const Button* manager_is_docked(Manager *m) {
```

Comment:
const? This sounds unnecessary. I see no reason why to make this needlessly specific. I mean, maybe someone actually wants to call this and make changes to the object returned one day, and I see no reason not to allow that.

"const" doesn't really make too much sense for things that are "alive objects", if you follow what i mean. I'd really just drop the "const" here.